piv/README.md

```diff
@@ -47,5 +45,11 @@ And use it to create `PivApplication` to select PIV applet on YubiKey
 ```
 
 ### Using the Demo Application <a name="using_demo"></a>
-Run demo app, select "PIV demo" pivot in navigation drawer. Plug in YubiKey and check what's current certificates you've got. You can generate new key and sign data with that key.
-Current demo doesn't allow you to import certificate from file, but it emulated import from pre-defined asset file. And exports to local cache file. Which also can be used to import certificate into another slot.
\ No newline at end of file
+1. Run demo app
+1. Select "PIV demo" pivot in navigation drawer
```

Comment:
Enumeration is wrong

Reply:
Markdown's numbered list feature automatically displays the correct numbering when using '1.'.  See https://guides.github.com/features/mastering-markdown/#syntax

Does this feature resolve the issue?

Reply:
Yes, thank you